profiling/profile.py

```diff
@@ -0,0 +1,83 @@
+"""
+Profiles conda processes using psutil plus a couple of built-in modules
+
+NOTICE: this currently only works on Unix-like systems (macOS and Linux)
+"""
+import resource
+import subprocess
+import sys
+import time
+
+import psutil
+
+#: Value used to convert bytes to megabytes
+BYTES_TO_MB = 1024 * 1024
+
+#: Value used to convert kilobytes to megabytes
+KILOBYTES_TO_MB = 1024
+
+
+def main():
+    """
+    Entry point for the profiling script
+    """
+
+    # Gather the network and time statistics before
+    net_io_stats_before = psutil.net_io_counters()
+    time_before = time.time()
+
+    command = [
+        "python", "-m",
+        # conda command
+        "conda", "create",
+        # settings
+        "--name", "profile-env",
+        "--channel", "conda-forge",
+        "--override-channels",
+        "--yes",
+        "--quiet",
+        # packages
+        "python",
+    ]
+    sub = subprocess.Popen(command)
+    sub.wait()
+
+    rusage = resource.getrusage(resource.RUSAGE_CHILDREN)
+
+    # Gather statistics after
+    net_io_stats_after = psutil.net_io_counters()
+    time_after = time.time()
```

Comment:
```suggestion
    time_after = time.monotonic_ns()```
    
Gives the time in nanoseconds, avoiding floating point errors or system clock updates.

> Return the value (in fractional seconds) of a monotonic clock, i.e. a clock that cannot go backwards. The clock is not affected by system clock updates. The reference point of the returned value is undefined, so that only the difference between the results of two calls is valid.